Tests/test_Compass.py

```diff
@@ -16,9 +17,7 @@ def setUp(self):
         file_dir = os.path.join("Compass")
         self.test_files = [
             os.path.join(file_dir, "comtest1"),
-            os.path.join(file_dir, "comtest2"),
-            os.path.join(file_dir, "comtest3"),
-            ]
+            os.path.join(file_dir, "comtest2")]
```

Comment:
Strange that we don't have a comtest3 folder but perhaps used to. Anyway, good spot.